src/core/ext/filters/client_channel/lb_policy/round_robin/round_robin.c

```diff
@@ -165,20 +165,20 @@ static size_t get_next_ready_subchannel_index_locked(
   if (GRPC_TRACER_ON(grpc_lb_round_robin_trace)) {
     gpr_log(GPR_INFO,
             "[RR: %p] getting next ready subchannel, "
-            "last_ready_subchannel_index=%zu",
-            p, p->last_ready_subchannel_index);
+            "last_ready_subchannel_index=%d",
+            p, (int)p->last_ready_subchannel_index);
```

Comment:
I think `int` is too small on some platforms.  If you need to cast it, then I suggest casting to an `unsigned long long` and using `%llu`.